services/node/testUtils/packageJsonVersionsTemplate.json

```diff
@@ -0,0 +1,662 @@
+{
```

Comment:
Could we trim this down to the minimal set of keys that are required? It'd really increase readability to only have the core pieces in here that are needed for the tests

Reply:
sure thing